### ISSUE TITLE
Dropout

### ISSUE THREAD

**Issue by Bleyddyn** [Bleyddyn/malpi#1]
Add support for dropout layers to the MalpiConvNet class.

**Comment by Bleyddyn** [comment]
Lots has changed since this issue was created, but current code in model_keras.py has support for dropout layers.
